plugin-flex-ts-template-v2/src/feature-library/dispositions/custom-components/Notes/Notes.tsx

```diff
@@ -0,0 +1,48 @@
+import { useState } from 'react';
+import { ITask, Template, templates } from '@twilio/flex-ui';
+import { TextArea } from '@twilio-paste/core/textarea';
+import { Label } from '@twilio-paste/core/label';
+import { HelpText } from '@twilio-paste/core/help-text';
+
+import { StringTemplates } from '../../flex-hooks/strings';
+
+export interface OwnProps {
+  task?: ITask;
+  notes: string;
```

Comment:
This value doesn't seem to be used. I noticed a regression where notes are no longer saved/restored when switching views, and this may be part of the reason.